pkg/cable/strongswan/strongswan.go

```diff
@@ -415,7 +415,7 @@ func (i *strongSwan) writeCharonConfig(path string) error {
 		return fmt.Errorf("error creating charon config file %q: %s", path, err)
 	}
 
-	if err = i.renderCharonConfigTemplate(f); err != nil {
+	if err := i.renderCharonConfigTemplate(f); err != nil {
```

Comment:
It's interesting that it complains about "sloppy reassign" here but not on L422 which does the same thing.

Reply:
Good point, I don't see why that be the case.